tests/lib/managers/terms-of-service-test.js

```diff
@@ -0,0 +1,571 @@
+/**
+ * @fileoverview Terms Of Service Tests
+ */
+'use strict';
+
+// ------------------------------------------------------------------------------
+// Requirements
+// ------------------------------------------------------------------------------
+var sinon = require('sinon'),
+	mockery = require('mockery'),
+	assert = require('chai').assert,
+	Promise = require('bluebird'),
+	leche = require('leche');
+
+var BoxClient = require('../../../lib/box-client');
+
+
+// ------------------------------------------------------------------------------
+// Helpers
+// ------------------------------------------------------------------------------
+var sandbox = sinon.sandbox.create(),
+	boxClientFake = leche.fake(BoxClient.prototype),
+	TermsOfService,
+	termsOfService,
+	TERMS_OF_SERVICE_ID = '1234',
+	TERMS_OF_SERVICE_USER_ID = '5678',
+	TERMS_OF_SERVICE_USER_STATUS_ID = '1234',
+	MODULE_FILE_PATH = '../../../lib/managers/terms-of-service';
+
+// ------------------------------------------------------------------------------
+// Tests
+// ------------------------------------------------------------------------------
+
+describe('TermsOfService', function() {
+
+	beforeEach(function() {
+		mockery.enable({
+			warnOnUnregistered: false
+		});
+		// Register Mocks
+		mockery.registerAllowable(MODULE_FILE_PATH, true);
+		// Setup File Under Test
+		TermsOfService = require(MODULE_FILE_PATH);
+		termsOfService = new TermsOfService(boxClientFake);
+	});
+
+	afterEach(function() {
+		sandbox.verifyAndRestore();
+		mockery.deregisterAll();
+		mockery.disable();
+	});
+
+	describe('get()', function() {
+		it('should make GET request to get terms of service info when called with ID', function() {
+			var response = {
+				type: 'terms_of_service',
+				id: '1234',
+				status: 'enabled',
+				enterprise: {},
+				tos_type: 'external',
+				text: 'TEST',
+				created_at: '2017-10-21T20:09:59-07:00',
+				modified_at: '2017-10-21T20:09:59-07:00'
+			};
+			sandbox.stub(boxClientFake, 'wrapWithDefaultHandler').returnsArg(0);
+			sandbox.mock(boxClientFake).expects('get').withArgs('/terms_of_services/1234')
+				.returns(Promise.resolve(response));
+			termsOfService.get(TERMS_OF_SERVICE_ID);
```

Comment:
For completeness, we should pass options here and ensure they get passed through to `get()`